Woof/Woof/View/OwnerCardView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+
+/// OwnerCardView is a SwiftUI view that displays the profile information in card view style of an Owner.
+struct OwnerCardView: View {
+    @ObservedObject var viewModel = OwnerCardViewModel()
+    @State private var isEditMode: Bool = false
+
+    var body: some View {
+        VStack {
+            HStack {
+                Image(systemName: "person.circle")
+                    .resizable()
+                    .frame(width: 40, height: 40)
+                    .foregroundColor(Color.App.purpleDark)
+
+                VStack(alignment: .leading, spacing: 5.0) {
+                    Group {
+                        TextField("Name", text: $viewModel.owner.name)
+                            .font(.system(size: 22, weight: .heavy))
+                            .foregroundColor(Color.App.white)
+
+                        TextField("Surname", text: $viewModel.owner.surname)
+                            .font(.system(size: 22, weight: .heavy))
+                            .foregroundColor(Color.App.white)
+
+                        TextField("Phone", text: $viewModel.owner.phone)
+                            .font(.system(size: 18, weight: .bold))
+                            .foregroundColor(Color.App.white)
+
+                        TextField("Adress", text: $viewModel.owner.address)
+                            .font(.system(size: 18, weight: .bold))
+                            .foregroundColor(Color.App.white)
+                    }
+                    .disabled(!isEditMode)
+                    .background(isEditMode ? Color.App.purpleDark : Color.clear)
+                    .foregroundColor(isEditMode ? Color.App.purpleDark : Color.App.purpleDark)
```

Comment:
This line does nothing.
Maybe you want to change the font color for all cases to white and remove duplicate lines of code from the group section?